README.md

```
@@ -18,7 +20,28 @@ Git ワークフロー自動化、GitHub レビュー管理、CI ログ分析の
 - **複数リポジトリ対応**: 親ディレクトリから実行すると変更のあるサブリポジトリを自動検出
 - **PRテンプレート対応**: `.github/pull_request_template.md` を自動検出・適用
 - **ブランチ自動作成**: デフォルトブランチにいる場合に自動でフィーチャーブランチを作成
-- **レビュー管理**: レビューコメントの妥当性検証と一括返信
+- **投稿ゲートの構造化**: レビュー（`review-pr`）と投稿（`post-review`）を別 Skill に分離し、確認前の投稿が起こらないようにする
+- **個別インラインコメント**: 指摘は該当行ごとのインラインコメントとして投稿
+
+### PR レビューの流れ
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)